### DISCUSSION TITLE
ADD A SECTION PULGIN

### DISCUSSION THREAD

**Discussion by SAKTHIVELMARIMUTHU** [GrapesJS/grapesjs#5701] [Q&A]
THE THING IS  section is like the one row or a column where we can add our default columns , image , vedio these kind of elements inside our section its is basically a container it hold the default elements the below code is a piece i wrote but unfortunatly i dont know what to place in content if its a image i place a image tag or a hedaing i place <h></h> or a paragraph means <p> so who to define a section it should be a row or a column 

Section1Plugin(editor) {
    editor.Blocks.add('section 1', {
      label: 'Section 1',
      category: 
      media: '<h>section</h>',
      content: '       ',
    });
  }

**Comment by mohamedsalem401** [comment]
The content should contain the actual HTML that should be added after dropping the block for example you can use something like

```
editor.Blocks.add('section-1', {
  label: 'Section 1',
  category: 'Media',
  media: '<div style="padding:5px; background:#eee;">Section</div>',
  content: `<section class="section">
              <div class="row">
                <div class="column"></div>
              </div>
            </section>`,
});
```
